tests/system_tests_interior_sync_up.py

```diff
@@ -136,13 +144,8 @@ def delay_timeout(self):
         self.add_receivers()
 
     def add_receivers(self):
-        if len(self.receivers) < self.count:
+        while len(self.receivers) < self.count:
```

Comment:
This changes the fundamental behavior of the test.  But from looking at the router source code there is no MAX_KEPT_DELTAS appearing anywhere so maybe the functionality this test was intended to exercise is no longer present?  @ted-ross does this test ring a bell?

Reply:
MAX_KEPT_DELTAS is no longer in the router.  When the "mobile" functionality moved from Python to C, that feature was removed.  MAX_KEPT_DELTAS is now effectively 1.